app/assets/sass/application-common.scss

```diff
@@ -74,8 +74,31 @@ li.task-item {
   padding-top: em(35);
   }
 
-  div.declaration-header {
+div.declaration-header {
   text-align: center;
   margin-top: 5px;
   margin-bottom: 5px;
+}
+
+#global-header .header-proposition #sign-out-header {
+  font-family: "nta", Arial, sans-serif;
+  text-transform: none;
+  font-size: 18px;
+  line-height: 1.2;
+  font-weight: bold;
+  float: right;
+  margin-right: 40px;
+  color: white;
+  text-decoration: none; }
+  @media (min-width: 641px) {
+    #global-header .header-proposition #sign-out-header {
+      font-size: 18px;
+      line-height: 1.25; } }
```

Comment:
Can we remove `font-family`, `text-transform`, `line-height` (from both the main rule and the media query) and `margin-right` so they are inherited from the cascade.

For the `font-size`, can we use `@include bold-16` for the main rule and `@include bold-24` for the media query. This will use the GOVUK defined mixins for the size and weight and follows the css for `#proposition-header` to make the `font-size` larger by specifying the mobile size in the main rule and overriding this with a larger size in the media query.

We should also use the `$white` GOVUK sass variable for the colour instead of specifying the value.

For example:

```
#global-header .header-proposition #sign-out-header {
  @include bold-16;
  float: right;
  color: $white;
  text-decoration: none; 
}
@media (min-width: 641px) {
  #global-header .header-proposition #sign-out-header {
    @include bold-24;
    line-height: 1.25; 
  } 
}
```

Can you also check that the base line of `#sign-out-header` is the same as `#proposition-header` for screens sizes smaller and larger than 641px.